_data/sites.json

```
@@ -1720,6 +1720,17 @@
         ]
     },
 
+    {
+        "name": "Bigstock",
+        "url": "https://support.bigstockphoto.com/s/article/How-do-I-close-my-Bigstock-contributor-account",
+        "difficulty": "hard",
+        "notes": "To delete your account, you must email support@bigstockphoto.com and request that your account be deleted.",
```

```suggestion
        "notes": "To delete your account, you must email tjem and request that your account be deleted.",
```

Since we have the e-mail field, I tend to simplify the notes.